DigitalOffice/Tests/UserServiceTests/RepositoriesTests/UserRepositoryTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Moq;
+using NUnit.Framework;
+using System.ComponentModel.DataAnnotations;
+using UserService.Database;
+using UserService.Database.Entities;
+using UserService.Mappers;
+using UserService.Models;
+using UserService.Repositories;
+using System;
+using System.Collections.Generic;
+
+namespace UserServiceUnitTests.RepositoriesTests
+{
+    public class UserRepositoryTests
+    {
+        UserServiceDbContext dbContext;
+        UserRepository repository;
+
+        [SetUp]
+        public void Initialization()
+        {
+            var options = new DbContextOptionsBuilder<UserServiceDbContext>()
+               .UseInMemoryDatabase(databaseName: "UserRepositoryLoginTest")
+               .Options;
+            dbContext = new UserServiceDbContext(options);
+
+            if (dbContext.Database.IsInMemory())
+            {
+                dbContext.Database.EnsureDeleted();
+            }
+        }
+
+        [Test]
+        public void ShouldFindUser()
+        {
+            var user = new DbUser
+            {
+                Email = "Example@mail.com",
+                Id = new Guid(),
+                FirstName = "",
+                LastName = "",
+                MiddleName = "",
+                Status = "",
+                PasswordHash = "",
+                AvatarFileId = new Guid(),
+
+                AchievementsIds = new List<DbUserAchievement>(),
+                CertificatesFilesIds = new List<DbUserCertificateFile>()
+            };
+
+            dbContext.Users.Add(user);
+            dbContext.SaveChanges();
```

Comment:
Move this logic to SetUp

Reply:
I think we only need to add a user in this specific test, why would we want a dbContetx to have that user when every test starts?

Reply:
Than move it to OneTimeSetUp